src/components/VFooter/VFooter.vue

```diff
@@ -113,52 +115,91 @@ export default defineComponent({
     const footerEl = ref<HTMLElement | null>(null)
     const { dimens: footerDimens } = useResizeObserver(footerEl)
     const variantNames = computed(() =>
-      Object.entries(SCREEN_SIZES)
-        .filter(([, val]) => footerDimens.value.width >= val)
+      [...Object.entries(SCREEN_SIZES), ['xs', 0]]
+        .find(([, val]) => footerDimens.value.width >= val)
         .map(([key]) => `footer-${key}`)
     )
 
+    const linkCount = computed<CSSProperties>(() => ({
+      '--link-count': Object.keys(allPages).length,
+    }))
+
     return {
       isContentMode,
       allPages,
 
       footerEl,
       variantNames,
+      linkCount,
     }
   },
 })
 </script>
 
 <style>
-.footer.footer-sm .nav-list {
-  @apply flex flex-row gap-8;
+.footer-xs .logo-and-links {
+  @apply flex-col;
+}
+.footer-xs .language {
+  width: 100% !important;
+}
+.footer-sm .logo-and-links {
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2, 1fr);
```

Comment:
IMO these CSS rules should be replaced `@apply`-ied Tailwind utilities.

Reply:
Done :)